pkg/minishift/cache/oc_caching_test.go

```diff
@@ -79,9 +86,27 @@ func setUp(t *testing.T) {
 	testOc = Oc{"v1.3.1", filepath.Join(testDir, "cache")}
 }
 
-func EnsureGitHubApiAccessTokenSet(t *testing.T) {
-	if githubutils.GetGitHubApiToken() == "" {
-		t.Skip("Skipping GitHub API based test, because no access token is defined in the environment.\n " +
-			"To run this test check https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/ and set for example MINISHIFT_GITHUB_API_TOKEN (see github.go).")
+func addMockResponses(mockTransport *minitesting.MockRoundTripper) {
+	testDataDir := filepath.Join(basepath, "..", "..", "..", "test", "testdata")
+
+	mockTransport.RegisterResponse("https://api.github.com/repos/openshift/origin/releases/tags/v1.3.1", &minitesting.CannedResponse{
```

Comment:
I noticed the same, but in this case it is just pinning on a known value, and not really the functionality provided by the version specified. it is mock and nothing more.